challenges/palindrome-detector.js

```diff
@@ -21,3 +21,17 @@
 */
 
 // YOUR CODE HERE
+
+function isPalindrome(str) {
+  var newStr = str.replace(/[^\w\s]/gi, '').replace(/ /g,'').toLowerCase(); //remove punctuation and spaces, convert to lowercas
```

Comment:
I'm assuming this is good use of Googling, but even so, I'm glad you were able to get this working!